plugins/techdocs-backend/src/techdocs/stages/prepare/azure.ts

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
We could take this opportunity to merge all these preparers into a 'git' preparer and just register that as different types on the preparers.

Reply:
Yeah. As I said in the description, I'm not sure if this could benefit from the work that @Rugvip has done in #2665 as well.

Reply:
Yea, we've discussed that a bit. We're going with this separate preparer now, but it's possible we'll try and take advantage of the work done in that PR soon.